vm/chef/cookbooks/apache2/recipes/mod-wsgi.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe 'apache2'
+
+package 'libapache2-mod-wsgi' do
```

Comment:
Is this one necessary if python2 has been removed? Since we install `libapache2-mod-wsgi-py3`

Reply:
In fact, I don't need any of apache2 changes. During the development I moved it to nginx. Reverting the apache changes.